db/schema_applier.cc

```diff
@@ -250,39 +250,45 @@ static future<std::set<sstring>> merge_keyspaces(distributed<service::storage_pr
     auto diff = difference(before, after, indirect_equal_to<lw_shared_ptr<query::result_set>>());
     auto sk_diff = difference(sk_before, sk_after, indirect_equal_to<lw_shared_ptr<query::result_set>>());
 
-    auto& created = diff.entries_only_on_right;
```

Comment:
> everything is updated. There is no indication that notification
> listeners depend on this behaviour.

Seems safe to me too. However, we better topologically sort notifications (create keyspace -> create table -> drop table -> drop keyspace). Out of scope here.

Reply:
Actually it seems to be not true, I am currently investigating but looks like one can't reorder notifications types as I am doing per commit and something else is needed to make all commits work one by one.